util/db/dbutil.go

```diff
@@ -132,6 +136,13 @@ func makeAccessorImpl(dbfilename string, readOnly bool, inMemory bool, params []
 	return db, err
 }
 
+// Clone creates a new Accessor with the same parameters as the original one
+func (db *Accessor) Clone() (Accessor, error) {
```

Comment:
This won't work correctly for writer accessor, since we can have only one writer accessor.

Reply:
yes, I think I'll limit it to RO and WAL in params to indicate intended use